plugins/scheduled_task/lib/MediaRepurposingHandler.php

```diff
@@ -6,6 +6,8 @@
  */
 class MediaRepurposingHandler implements kObjectDataChangedEventConsumer
 {
+    	const IGNORE_MRP_RESET_ON_METADATA_UPDATE = "ignoreMRPResetOnMetadataUpdate";

```

Comment:
change to something like:
[excluded_MRP_ids_from_metadata_update]

0 = MRP ID1
1 = MRP ID2

and remove MR_PROFILES const

Reply:
done